pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.mycompany</groupId>
+    <groupId>cz.muni.fi.pa165.pokemoni</groupId>
     <artifactId>TournamentManager</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>jar</packaging>
```

Comment:
Netbeans gives this error:
'packaging' with value 'jar' is invalid
better replace with 'pom'

Reply:
How about war, war is basically a zip that is ready to eb deployed on applicatin server?
Our application will have to be built into war at the end of the project.